src/main/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/ObjectionService.java

```diff
@@ -122,20 +122,20 @@ public Objection getObjection(String requestId, String objectionId) throws Objec
         String attachmentId = response.getFileId();
         if (StringUtils.isBlank(attachmentId)) {
             throw new ServiceException("No file id returned from file upload");
-        } else {
-            Attachment attachment = createAttachment(file, attachmentId);
-            Objection objection = objectionRepository.findById(objectionId).orElseThrow(
-                    () -> new ObjectionNotFoundException(String.format(OBJECTION_NOT_FOUND_MESSAGE, objectionId))
-            );
-            objection.addAttachment(attachment);
+        }
```

Comment:
this is just formatting, removed the else block to reduce the levels of nesting (similar to the throw block above this one)